src/sentry/web/frontend/js_sdk_dynamic_loader.py

```diff
@@ -13,6 +34,71 @@ class JavaScriptSdkDynamicLoader(BaseView):
     def determine_active_organization(self, request: Request, organization_slug=None) -> None:
         pass
 
+    def _get_context(self, key: ProjectKey) -> Tuple[LoaderContext, Optional[str], Optional[str]]:
+        """Sets context information needed to render the loader"""
+        if not key:
+            return ({}, None, None)
+
+        sdk_version = get_browser_sdk_version(key)
+
+        bundle_kind_modifier = self._get_bundle_kind_modifier(key)
+
+        sdk_url = ""
+        try:
+            sdk_url = settings.JS_SDK_LOADER_DEFAULT_SDK_URL % (sdk_version, bundle_kind_modifier)
+        except TypeError:
+            sdk_url = ""  # It fails if it cannot inject the version in the string
+
+        return (
+            {
+                "config": {
+                    "dsn": key.dsn_public,
+                    "jsSdkUrl": sdk_url,
+                    "publicKey": key.public_key,
+                }
+            },
+            sdk_version,
+            sdk_url,
+        )
+
+    def _get_bundle_kind_modifier(self, key: ProjectKey) -> str:
+        """Returns a string that is used to modify the bundle name"""
+        bundle_kind_modifier = ""
+
+        if get_dynamic_sdk_loader_option(key, DynamicSdkLoaderOption.HAS_PERFORMANCE):
+            bundle_kind_modifier += ".tracing"
+
+        if get_dynamic_sdk_loader_option(key, DynamicSdkLoaderOption.HAS_REPLAY):
+            bundle_kind_modifier += ".replay"
+
+        # TODO(abhi): Right now this loader only supports returning es6 JS bundles.
+        # We may want to re-evaluate this.
+        # if es5
+        # bundle_kind_modifier += ".es5"
+
+        if get_dynamic_sdk_loader_option(key, DynamicSdkLoaderOption.HAS_DEBUG):
+            bundle_kind_modifier += ".debug"
+
+        return bundle_kind_modifier
+
     def get(self, request: Request, public_key: str, minified: str) -> Response:
         """Returns a JS file that dynamically loads the SDK based on project settings"""
-        return super().get(request)
+        key = None
+        try:
+            key = ProjectKey.objects.get_from_cache(public_key=public_key)
+        except ProjectKey.DoesNotExist:
+            pass
+        else:
+            key.project = Project.objects.get_from_cache(id=key.project_id)
+
+        # TODO(abhi): Return more than no-op template
+        tmpl = "sentry/js-sdk-loader-noop.js.tmpl"
+
+        context, sdk_version, sdk_url = self._get_context(key)
+
+        response = render_to_response(tmpl, context, content_type="text/javascript")
+
+        response["Access-Control-Allow-Origin"] = "*"
```

Comment:
Do you also want to set [Cross-Origin-Resource-Policy](https://developer.mozilla.org/en-US/docs/Web/HTTP/Cross-Origin_Resource_Policy) for more restrictive cross-origin policy sites?

Reply:
We want the cross-origin policy to be as broad as possible, we want any site to be able to load the loader.

Reply:
That's what I thought, you'll want `Cross-Origin-Resource-Policy: cross-origin` then.